sormas-api/src/main/resources/strings.properties

```diff
@@ -15,155 +15,150 @@
 # You should have received a copy of the GNU General Public License
```

Comment:
You have to revert the changes to this file. We'll run into heavy merge problems with the I18n branch and during a potential hotfix like this.